docs/references/annotations.md

```diff
@@ -4,9 +4,11 @@ Kong Ingress Controller uses some annotations to configure Ingress resources.
 
 It supports the following annotations:
 
-- [`kubernetes.io/ingress.class`](#kubernetesioingressclass)
-- [`plugins.konghq.com`](#pluginskonghqcom)
-- [`configuration.konghq.com`](#configurationkonghqcom)
+| Annotation name | Description | Guide |
+|-----------------|-------------|-------|
+| [`kubernetes.io/ingress.class`](#kubernetesioingressclass) | multiple ingres|
+| [`plugins.konghq.com`](#pluginskonghqcom) | multiple ingres|
+| [`configuration.konghq.com`](#configurationkonghqcom) | |
```

Comment:
What is going in the "guide" column?

Reply:
Thank you for catching this.
I've added this in the last commit I pushed.